Training/TPU-Trillium/Llama3-405B-PyTorch/host.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+DOCKER_IMAGE=us-central1-docker.pkg.dev/deeplearning-images/reproducibility/pytorch-xla/llama3-405b:nightly-sep28
+
+worker_id=$(curl -s "http://metadata.google.internal/computeMetadata/v1/instance/attributes/agent-worker-number" -H 'Metadata-Flavor: Google')
+
+cat >> /dev/null <<EOF
```

Comment:
what is this doing?

Reply:
I don't really know -- I copied it from existing scripts. I know at a high level this runs the training command in a docker container.